app/views/social/index.html

```diff
@@ -20,14 +20,12 @@ <h1>Testing social media button integration</h1>
 
       <p>PTC 190 - Once a user has completed the sign-up journey they can easily share the link to the service through their social media channels.</p>
 
-      <p><strong>We have 3 options</strong></p>
-
-      <h2>Share buttons</h2>
+      <h2>Social Media Share buttons</h2>
 
       <ul>
-        <li><a href="http://localhost:3000/social/normal">Option 1: Normal</a></li>
-        <li><a href="http://localhost:3000/social/inset">Option 2: Inset</a></li>
-        <li><a href="http://localhost:3000/social/separate">Option 3: Separate</a></li>
+        <li><a href="https://permission-to-contact-proto.herokuapp.com/social/normal">Option 1: Normal</a></li>
```

Comment:
Maybe remove the full URL and have the link relative to the root, eg <a href="/social/normal">